<ui version="4.0" >
 <class>CDDAHeaderDataWidgetUI</class>
 <widget class="QWidget" name="CDDAHeaderDataWidgetUI" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>282</width>
    <height>372</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" colspan="2" >
    <widget class="QCheckBox" name="checkBox_various" >
     <property name="text" >
      <string>Various Artists</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" >
    <widget class="QCheckBox" name="checkBox_multicd" >
     <property name="text" >
      <string>Multi CD</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3" >
    <spacer name="horizontalSpacer" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>62</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label_artist" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Artist:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3" >
    <widget class="KLineEdit" name="klineedit_artist" />
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label_title" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Title:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3" >
    <widget class="KLineEdit" name="klineedit_title" />
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="label_cdnum" >
     <property name="text" >
      <string>CD Number:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="3" >
    <widget class="KIntSpinBox" name="kintspinbox_cdnum" >
     <property name="minimum" >
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Track Offset:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="3" >
    <widget class="KIntSpinBox" name="kintspinbox_trackoffset" />
   </item>
   <item row="5" column="0" >
    <widget class="QLabel" name="label_genre" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Genre:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="3" >
    <widget class="KComboBox" name="kcombobox_genre" >
     <property name="editable" >
      <bool>true</bool>
     </property>
     <property name="autoCompletion" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0" >
    <widget class="QLabel" name="label_year" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Year:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="3" >
    <widget class="KIntSpinBox" name="kintspinbox_year" >
     <property name="minimum" >
      <number>-3000</number>
     </property>
     <property name="maximum" >
      <number>3000</number>
     </property>
     <property name="value" >
      <number>2004</number>
     </property>
    </widget>
   </item>
   <item row="7" column="0" >
    <widget class="QLabel" name="label_extdata" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Additional
Information:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="3" >
    <widget class="KTextEdit" name="ktextedit_extdata" />
   </item>
   <item row="8" column="0" >
    <widget class="QLabel" name="label_discid" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Disc ID:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1" colspan="3" >
    <widget class="QLabel" name="label_discid_2" >
     <property name="text" >
      <string>0x00000000</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KIntSpinBox</class>
   <extends>QSpinBox</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KTextEdit</class>
   <extends>QTextEdit</extends>
   <header>ktextedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>checkBox_various</tabstop>
  <tabstop>checkBox_multicd</tabstop>
  <tabstop>klineedit_artist</tabstop>
  <tabstop>klineedit_title</tabstop>
  <tabstop>kintspinbox_cdnum</tabstop>
  <tabstop>kintspinbox_trackoffset</tabstop>
  <tabstop>kcombobox_genre</tabstop>
  <tabstop>kintspinbox_year</tabstop>
  <tabstop>ktextedit_extdata</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
